pyproject.toml

```diff
@@ -141,7 +144,7 @@ omit = [
 # https://flit.pypa.io/en/latest/pyproject_toml.html#sdist-section
 # See also: https://github.com/pypa/flit/issues/565
 [tool.flit.sdist]
-include = []
+include = ["src/macaron/resources/schemastore/*"]
```

Comment:
`flit` should include all the files under `src/macaron` by default. I wonder if you faced any issues and needed to add this path explicitly?

Reply:
`flit` only includes files that are in the git repo. These files are downloaded as part of `make setup`, so need to be included explicitly.

Reply:
Hm, we also generate for example the `bashparser` binary during `make setup` and store it under `src/macaron/resources`, which gets included in the package. Is there a difference here?

Reply:
Looks like the "only files in the git repo" behaviour only applies to the sdist, for the wheel it does include all files under the directory. So built binaries such as `bashparser` are packaged into the wheel but not the sdist. I suppose that makes sense, as from my limited understanding, the wheel is supposed to be the fully prebuilt ready-to-use package and the sdist is supposed to be the source distribution, though supposedly you are also supposed to be able to install the package from the sdist, which I don't think you can do with ours because it doesn't contain any build script to produce the binaries, so I'm not sure what the purpose of our sdist is if you have to use the wheel for installing. 

The original issue that I encountered trying to build the package with `flit` was that it wouldn't build if there were untracked files in the repo, I fixed that by adding the schemastore files to gitignore but I also just assumed that they were supposed to be included in the sdist, but now I'm not so sure, we could just leave them out as well since they are not our source code (regardless, they will be included in the wheel).

Reply:
Yes, that makes sense. If someone wants to build the Macaron Python package from source, just using the `sdist` will not be enough and they need to use the `Makefile`. But we haven't made an attempt to allow automatically build the package from source using `pip install`. If/when we release Macaron on PyPI, we should look into this.